base/linalg/lu.jl

```diff
@@ -31,6 +31,35 @@ end
 input `A`, instead of creating a copy. An [`InexactError`](@ref)
 exception is thrown if the factorization produces a number not representable by the
 element type of `A`, e.g. for integer types.
+
+# Examples
+```jldoctest
+julia> A = [4 3; 6 3]
+2×2 Array{Int64,2}:
+ 4  3
+ 6  3
+
+julia> lufact!(A)
+ERROR: InexactError: convert(Int64, 0.6666666666666666)
```

Comment:
I would switch around this too, I think it is nicer with a working example first. Perhaps
```julia
julia> A = [4. 3.; 6. 3.]
2×2 Array{Float64,2}:
 4.0  3.0
 6.0  3.0

julia> lufact!(A)
Base.LinAlg.LU{Float64,Array{Float64,2}}
L factor:
2×2 Array{Float64,2}:
 1.0       0.0
 0.666667  1.0
U factor:
2×2 Array{Float64,2}:
 6.0  3.0
 0.0  1.0

julia> B = [4 3; 6 3]
2×2 Array{Int64,2}:
 4  3
 6  3

julia> lufact!(B)
ERROR: InexactError: convert(Int64, 0.6666666666666666)
Stacktrace:
[...]
```